.github/actions/deploy-ubuntu/action.yml

```diff
@@ -98,7 +98,22 @@ runs:
         # $SUDO apt-key adv --keyserver keyserver.ubuntu.com --recv-keys BA6932366A755776
         $SUDO apt-get update
         $SUDO apt-get -y install gcc-multilib g++-multilib gfortran-multilib python3 python2.7 python3-minimal python2.7-minimal rpm libasound2-dev:$ARCH freeglut3-dev:$ARCH libfontconfig-dev:$ARCH libgtk2.0-dev:$ARCH libusb-dev:$ARCH libusb-1.0-0-dev:$ARCH libffi-dev:$ARCH libbz2-dev:$ARCH zlib1g-dev:$ARCH libxcb1-dev:$ARCH
-        $SUDO apt-get -y install pkg-config ccache clang $TOOLCHAIN openjdk-8-jdk ant python2 python3-pip swig git file wget unzip tar bzip2 gzip patch autoconf-archive autogen automake cmake make libtool bison flex perl nasm ragel curl libcurl4-openssl-dev libssl-dev libffi-dev libbz2-dev zlib1g-dev rapidjson-dev
+        $SUDO apt-get -y install pkg-config ccache clang $TOOLCHAIN openjdk-8-jdk ant python2 python3-pip swig git file wget unzip tar bzip2 gzip patch autoconf-archive autogen automake cmake make libtool bison flex perl nasm ragel curl libcurl4-openssl-dev libssl-dev libffi-dev libbz2-dev zlib1g-dev rapidjson-dev libdrm-dev
+        # install vulkan sdk
+        echo Installing Vulkan-SDK
+        wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
+        $SUDO wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.268-focal.list https://packages.lunarg.com/vulkan/1.3.268/lunarg-vulkan-1.3.268-focal.list
+        $SUDO apt update
+        $SUDO apt-get install -y libvulkan1 libvulkan-dev
+        $SUDO apt install vulkan-sdk
```

Comment:
We can probably put that on the same line?

Reply:
So the installation of the vulkan sdk would look like that

```
        # install vulkan sdk
        echo Installing Vulkan-SDK
        curl -s https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
        $SUDO curl -s -o /etc/apt/sources.list.d/lunarg-vulkan-1.3.268-focal.list https://packages.lunarg.com/vulkan/1.3.268/lunarg-vulkan-1.3.268-focal.list
        $SUDO apt update && $SUDO apt-get install -y libvulkan1 libvulkan-dev && $SUDO apt install vulkan-sdk
```

Or do you want the the stuff in one line? I just put the apt/ apt-get install in one line for now:

If okay I will push but this will trigger again the checks, is that a problem? I'm currently building my test branch to check if everything works with these changes

Reply:
Ah yes that will for sure work, it was a leftover from my trial and error. Is it okay now?

```
        # install vulkan sdk
        echo Installing Vulkan-SDK
        curl -s https://packages.lunarg.com/lunarg-signing-key-pub.asc | $SUDO apt-key add -
        $SUDO curl -s -o /etc/apt/sources.list.d/lunarg-vulkan-1.3.268-focal.list https://packages.lunarg.com/vulkan/1.3.268/lunarg-vulkan-1.3.268-focal.list
        $SUDO apt-get update && $SUDO apt-get install -y libvulkan1 libvulkan-dev vulkan-sdk
```

Reply:
Yes, that looks alright

Reply:
So is it okay to push my changes?

Reply:
Okay done :)